test/runtime/runtime-php.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { execCommand } from '../blackbox/utils/Utils';
+import path from 'path';
+
+jest.setTimeout(50000);
+
+test('PHP runtime testing', async () => {
+  const compileCommand = `cd ${path.resolve(
+    __dirname,
+    './runtime-php'
+  )} && composer install --no-interaction --quiet && composer test`;
```

Comment:
There is one problem with this command, as PHPUNIT does always output information and its taken by execCommand as error, but it isnt